test/smoke-fails/omptest-testsuite-emi-tracing/omptest_testsuite_emi_tracing.cpp

```diff
@@ -0,0 +1,231 @@
+#include "OmptTester.h"
+
+using namespace omptest;
+using namespace internal;
+
+TEST(DisabledSuite, DISABLED_uut_target_disabled) {}
```

Comment:
Do we want to put in something like `assert(false && "This test should not have been run")` so that we see if it is in fact not disabled?

Reply:
Yes, I think that's a good idea -- agreed.